The program currently looks in $out/etc for configuration. `/etc` might be a better default.

```console
$ mpdas
(17:37:41) [INFO] Cannot parse config file (/nix/store/xcfmrnnqfi50hiaymxixcfvivdfqnlkw-mpdas-0.4.4/etc/mpdasrc).
(17:37:41) [ERROR] AudioScrobbler username or password not set
$ strings $(which mpdas) | grep etc
/nix/store/xcfmrnnqfi50hiaymxixcfvivdfqnlkw-mpdas-0.4.4/etc
```

--- a/pkgs/tools/audio/mpdas/default.nix
+++ b/pkgs/tools/audio/mpdas/default.nix
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ mpd_clientlib curl ];
 
-  makeFlags = [ "DESTDIR=" "PREFIX=$(out)" ];
+  makeFlags = [ "CONFIG=/etc" "DESTDIR=" "PREFIX=$(out)" ];
 
   meta = with stdenv.lib; {
     description = "Music Player Daemon AudioScrobbler";

